src/todolistcheckcommand.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module list/todolistcheckedcommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+
+const attributeKey = 'todoListChecked';
+
+/**
+ * @extends module:core/command~Command
+ */
+export default class TodoListCheckCommand extends Command {
+	/**
+	 * @param {module:core/editor/editor~Editor} editor
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		this._selectedElements = [];
+
+		/**
+		 * Flag indicating whether the command is active. The command is active when the
+		 * {@link module:engine/model/selection~Selection#hasAttribute selection has the attribute} which means that:
+		 *
+		 * * If the selection is not empty &ndash; That the attribute is set on the first node in the selection that allows this attribute.
+		 * * If the selection is empty &ndash; That the selection has the attribute itself (which means that newly typed
+		 * text will have this attribute, too).
```

Comment:
Looks like you copy it and forget to update ;)

Reply:
It's possible :) I need to check all API docs. It's on todo list.

Reply:
Fixed.